code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -219,8 +219,7 @@
 	fire_sound = 'sound/weapons/lasercannonfire.ogg'
 	origin_tech = Tc_COMBAT + "=4;" + Tc_MATERIALS + "=3;" + Tc_POWERSTORAGE + "=3"
 	projectile_type = "/obj/item/projectile/beam/heavylaser"
-
-	fire_delay = 2
+	fire_delay = 20 // 2 seconds fire delay
 
```

Comment:
Change to '2 SECONDS' rather than 20

Reply:
I was thinking of that but 
A) It's a bit unnecessary when all that is needed is a 0
B) The real reason I didn't use it is because it didn't show up colored so maybe it didn't work for that instance